### PR TITLE
Update AWS EKS deployment docs.

### PR DESCRIPTION
**Pull request by praveenkumarct**
- Provide scripts to create AWS Cluster.
- Update docs.
- Fix Failing flaky [test](https://github.com/commercetools/commercetools-subscriptions/pull/48/files#diff-2c47e9c349ddebdd9fbdfeb8d7fe42e213d3aa88997dcf1ffd37bc25343ac8b4).

### CODE REVIEW

File: deployment-examples/aws/k8s/README.md [hunk 1]
```diff
@@ -34,19 +31,23 @@ Once all the prerequisites have been installed and created, update the repositor
 
 For any changes for cronjob, configure the properties in [values.yaml](./values.yaml).
 
-## Create release.
+## Creating AWS cluster (optional)
```
Comment (lojzatran) [inline deployment-examples/aws/k8s/README.md]: So you don't need a cluster to deploy to AWS?

File: deployment-examples/aws/k8s/README.md [hunk 2]
```diff
@@ -34,19 +31,23 @@ Once all the prerequisites have been installed and created, update the repositor
...
+
+Configure all the required environment variables in the bash script [`create_aws_cluster`](create_aws_cluster.sh). After that, execute the script file.
```
Comment (lojzatran) [inline deployment-examples/aws/k8s/README.md:36]: In the part `### Environment variables` above you wrote this:
```
update the repository secrets for the following fields in the repository.
```
I don't understand that sentence. Please update that sentence. For example here you have `Configure all the required environment variables in the bash script create_aws_cluster` which is clear.

File: deployment-examples/aws/k8s/create_aws_cluster.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+
+## edit these environment variables
+## ----------------------
+export AWS_REGION_CODE="us-west-2"
```
Comment (lojzatran) [inline deployment-examples/aws/k8s/create_aws_cluster.sh:7]: Here you have `AWS_REGION_CODE` and `AWS_CLUSTER_NAME` as fixed texts. In [deploy_to_eks.sh](https://github.com/commercetools/commercetools-subscriptions/pull/48/files#diff-2ab7b61674d59ff1e35679d77b0646d0f3ed6745b3301100203b6ebec89e2dba) you have env vars taken from somewhere as variables. Please unite the style and update the doc accordingly.

File: deployment-examples/aws/k8s/deploy_to_eks.sh [hunk 4]
```diff
@@ -23,7 +29,7 @@ printf "\n- Build and push docker images to AWS ECR\n"
 ECR_PATH=$AWS_ECR_PATH
 IMAGE_FULL_NAME=${ECR_PATH}/${IMAGE_BASE_NAME}
 
-aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin $ECR_PATH
+aws ecr get-login-password --region ${REGION_CODE} | docker login --username AWS --password-stdin $ECR_PATH
```
Comment (lojzatran) [inline deployment-examples/aws/k8s/deploy_to_eks.sh:32]: Why the username is always `AWS`?

**Review by lojzatran — CHANGES_REQUESTED** [review]

File: deployment-examples/aws/k8s/README.md (hunk 1 above)
Comment (praveenkumarct) [reply deployment-examples/aws/k8s/README.md]: https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/3041202de8f13db3d80bb37b0ecca11c6888d5f4

It is needed only for the first time.
I corrected the sentence.

File: deployment-examples/aws/k8s/deploy_to_eks.sh (hunk 4 above)
Comment (praveenkumarct) [reply deployment-examples/aws/k8s/deploy_to_eks.sh:32]: It is how AWS wishes to have. This is the auto generated command from AWS, you can check this in ECR repository.
Related doc link: https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html

File: deployment-examples/aws/k8s/README.md (hunk 2 above)
Comment (praveenkumarct) [reply deployment-examples/aws/k8s/README.md:36]: https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/10191179ef546c4356d12fa916e19262eb5227cf

File: deployment-examples/aws/k8s/create_aws_cluster.sh (hunk 3 above)
Comment (praveenkumarct) [reply deployment-examples/aws/k8s/create_aws_cluster.sh:7]: https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/10191179ef546c4356d12fa916e19262eb5227cf

File: deployment-examples/aws/k8s/README.md [hunk 5]
```diff
@@ -9,44 +9,45 @@ The main goal of this deploy process is to build and publish the new docker imag
...
-describe the new release and publish it.
+```bash
+./create_aws_cluster.sh
+```
 
-Bash scripts have been created to build and deploy docker image to AWS EKS as a cronjob. CD github actions job will run these bash scripts to deploy the new version to AWS.
+## Deploy to AWS cluster
 
-> In the deployment script, the [cron-job](https://github.com/commercetools/k8s-charts/tree/master/charts/cronjob) helm chart is used.
+Configure all the required environment variables in the bash script [`deploy_to_eks.sh`](deploy_to_eks.sh) and in the helm [values](values.yaml) file. After that, execute the script file.
```
Comment (lojzatran) [inline deployment-examples/aws/k8s/README.md]: Why do you need to configure twice, once in the bash script and once in the helm file? Can't you do it once only, either helm file or the bash script file?

Comment (praveenkumarct) [reply deployment-examples/aws/k8s/README.md]: https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/3e991b3f764a0379429497d4d64e9972c79f7807

**Review by lojzatran — APPROVED** [review]

**Pull request merged**